recipe/meta.yaml

```diff
@@ -9,26 +9,27 @@ source:
   sha256: 6caad9786055cb1fa22b4a365c1775816b876f91966481765d7d50e9f0dd35cc
 
 build:
-  number: 0
-  noarch: python
+  number: 1
   script: {{ PYTHON }} -m pip install . -vv
   entry_points:
     - sphinx-build = sphinx.cmd.build:main
     - sphinx-quickstart = sphinx.cmd.quickstart:main
     - sphinx-apidoc = sphinx.ext.apidoc:main
     - sphinx-autogen = sphinx.ext.autosummary.generate:main
+  skip: true  # [py<36]
 
 requirements:
   host:
-    - python >=3.6
+    - python
     - pip
     - babel
   run:
-    - python >=3.6
+    - python
     - alabaster >=0.7,<0.8
     - babel >=1.3
     - docutils >=0.14,<0.18
     - imagesize
+    - importlib-metadata >=4.4  # [py<310]
```

Comment:
note, this _replaces_ `setuptools`, so that can be removed below... c/f https://github.com/sphinx-doc/sphinx/pull/10007